examples/regular/models/creditcard_ctgan.py

```diff
@@ -40,7 +39,7 @@
 #----------------------------
 
 batch_size = 500
-epochs = 500+1
+epochs = 2+1
```

Comment:
Is it intended for the example to train the network with such a small number of epochs?